### PR TITLE
Opposite Activity Check

### PR DESCRIPTION
**Pull request by tgxn**
As per https://github.com/tgxn/lemmy-explorer/issues/75

Added check for really low user count compared to total comments+posts

### CODE REVIEW

**Comment by tgxn** [comment]
Filtering:
image

**Comment by db0** [comment]
Such small fractions are very difficult to visualize natively for humans natively. Can you reverse the calculation like so

`local_activity / node["total_users"]  > activity_suspicion_low`

Then we pass an integer which translates to "over this number of comments/post per user, is suspicious"

File: fediseer/observer.py [hunk 1]
```diff
@@ -64,10 +64,17 @@ def retrieve_suspicious_instances(activity_suspicion = 20, active_suspicious = 5
...
             if local_activity == 0:
                 local_activity= 1
+
+            # posts+comments could be much lower than total users
             if node["total_users"] / local_activity > activity_suspicion:
                 is_bad = True
                 # print(node)
 
+            # posts+comments could be much higher than total users
+            if local_activity / node["total_users"] < activity_suspicion_low:
```
Comment (db0) [inline fediseer/observer.py:74]: needs to be greater than

Comment (tgxn) [reply fediseer/observer.py]: It was a late night! 😴 I hope that's better now :D

**Pull request merged**